util/cron/test-perf.cray-cs-hdr.arkouda.bash

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Run arkouda testing on a cray-cs with HDR IB
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs-hdr'
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs-hdr.arkouda"
+
+# setup arkouda
+source $CWD/common-arkouda.bash
+export ARKOUDA_NUMLOCALES=16
+
+module list
```

Comment:
is there some voodoo behind this, or is it just to make it easier to debug a failure?

Reply:
No voodoo, just debugging -- https://github.com/chapel-lang/chapel/pull/16678